script/projects.js

```diff
@@ -16,18 +16,48 @@ Project.prototype.toHtml = function() {
   return templateRender(this);
 };
 
-//creates array of project objects
-projectDataSet.forEach(function(projectObject) {
-  projects.push(new Project(projectObject));
-});
-
-//makes list of projects in nav
-projects.forEach(function(myNewProjectObject) {
-  $('#project-list').append('<li>' + myNewProjectObject.title + '</li><hr>').hide();
-  $('#projects').append(myNewProjectObject.toHtml());
-
-});
+//looks for localStorage and/or saves to
+Project.fetchAll = function() {
+  if (localStorage.rawData) {
+    var data = JSON.parse(localStorage.rawData);
+  } else {
+    //ajax call
+    ($.getJSON('data/raw.json').then(function(data){
```

Comment:
Your $.getJSON call does not need to be inside of parenthesis. I would recommend removing them